wiki/Tournaments/Official_support/en.md

```diff
@@ -153,11 +153,9 @@ With regards to staff roles, the following rules apply:
 
-Users under an active tournament ban are expected to disclose their tournament ban status to any officially supported tournament they intend to help with. They may be enlisted as streamers, commentators, or graphic designers at the host's discretion, but we encourage careful consideration of such choices as said individuals have already infringed upon the rules once (or more). They may not act as any other staff role UNLESS the host requests an exemption for them when sending in their initial request for support. The account support team will then assess these on a case-by-case basis with the user's history in mind.
+Users under an active [tournament ban](/wiki/Help_centre/Tournament_sanctions#tournament-bans) or [staffing ban](/wiki/Help_centre/Tournament_sanctions#staffing-bans) are expected to disclose their ban status to any officially supported tournament they intend to help with. They may be enlisted as streamers, commentators, or graphic designers at the host's discretion, but we encourage careful consideration of such choices as said individuals have already infringed upon the rules once (or more). They may not act as any other staff role UNLESS the host requests an exemption for them when sending in their initial request for support. The account support team will then assess these on a case-by-case basis with the user's history in mind.
 
-At the support team's discretion, staff members whose conduct directly results in a tournament losing its official support may not fulfil the same roles in, nor be the host or admin of, another officially-supported tournament for at least 4 months. In the event a staffing ban is applied to a user, they will be informed directly through the [tournament team](mailto:tournaments@ppy.sh).
-
-At the support team's discretion, organisers who make major mistakes that break eligibility rules or expectations for official support may be subject to a hosting ban. Hosting bans may be used in place of revoking official support when a tournament would otherwise be ineligible for support while maintaining competitive integrity. While under a hosting ban, users may not be a primary organiser or admin of any officially-supported tournament. They may, however, engage in other roles which do not involve organisation.
+At the support team's discretion, organisers who make major mistakes that break eligibility rules or expectations for official support may be subject to a [hosting ban](/wiki/Help_centre/Tournament_sanctions#hosting-bans). Hosting bans may be used in place of revoking official support when a tournament would otherwise be ineligible for support while maintaining competitive integrity. While under a hosting ban, users may not be a primary organiser or admin of any officially-supported tournament. They may, however, engage in other roles which do not involve organisation. Less severe mistakes may lead to a [hosting probation](/wiki/Help_centre/Tournament_sanctions#hosting-probations) being applied instead.
```

Comment:
```suggestion
At the support team's discretion, organisers who make major mistakes that break eligibility rules or expectations for official support may be subject to a [hosting ban](/wiki/Help_centre/Tournament_sanctions#hosting-bans). Hosting bans may be used in place of revoking official support when a tournament would otherwise be ineligible for support while maintaining competitive integrity. While under a hosting ban, users may not be a primary organiser or admin of any officially supported tournament. They may, however, engage in other roles which do not involve organisation. Less severe mistakes may lead to a [hosting probation](/wiki/Help_centre/Tournament_sanctions#hosting-probations) being applied instead.
```